app/controllers/events.js

```diff
@@ -0,0 +1,65 @@
+const bodybuilder = require('bodybuilder');
+const { get, isEmpty } = require('lodash');
+const uuid = require('uuid');
+
+const client = require('../services/elasticsearch');
```

Comment:
please reanme client to smth more convinient

Reply:
e.g elasticsearchClient?

Reply:
I'm prefer camelCase naming... elasticSearchClient and elasticSearch file name

Reply:
I [renamed](https://github.com/inst-event-emitter/eventemitter-backend/pull/10/commits/f14759b667e1a373589dbdf79e85b03fdada47ee) elastic server